src/containers/index.js

```diff
@@ -53,4 +53,3 @@ export { default as Trigger } from './Trigger';
 export { default as Triggers } from './Triggers';
 export { default as TriggerTemplate } from './TriggerTemplate';
 export { default as TriggerTemplates } from './TriggerTemplates';
-export { default as YAMLEditor } from './YAMLEditor';
```

Comment:
Had to remove this to let the chunk splitting work properly, vite helpfully pointed this out:

```
(!) dashboard/src/containers/YAMLEditor/index.js is dynamically imported by dashboard/src/containers/CreatePipelineRun/CreatePipelineRun.jsx, dashboard/src/containers/CreateTaskRun/CreateTaskRun.jsx but also statically imported by dashboard/src/containers/index.js, dynamic import will not move module into another chunk.
```